### PR TITLE
Add lldk streaming

### PR DESCRIPTION
**Pull request by IonutMuthi**

### CODE REVIEW

File: examples/LLDKDemo.m [hunk 1]
```diff
@@ -0,0 +1,45 @@
+% set the bord uri
```
Comment (ribdp) [inline examples/LLDKDemo.m:1]: Typo: bord->board

File: examples/LLDKDemo.m [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+uri = 'ip:10.48.65.161';
```
Comment (ribdp) [inline examples/LLDKDemo.m:2]: Nit: Should this be changed to ip:192.168.2.1?

File: examples/LLDKDemo.m [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+% create a sinus signal
```
Comment (ribdp) [inline examples/LLDKDemo.m:4]: Nit: Should we expand 'sinus' to 'sinusoid'?

File: test/runHWTests.m [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+import matlab.unittest.plugins.XMLPlugin
+import matlab.unittest.plugins.DiagnosticsValidationPlugin
+import matlab.unittest.parameters.Parameter
+import matlab.unittest.plugins.ToUniqueFile;
+import matlab.unittest.plugins.TAPPlugin;
+import matlab.unittest.constraints.ContainsSubstring;
+import matlab.unittest.selectors.HasName;
+import matlab.unittest.selectors.HasProcedureName;
+
+switch board
```
Comment (ribdp) [inline test/runHWTests.m:15]: The boards and tests referred to, here, aren't relevant to the Precision Toolbox. We can scrap those cases, perhaps? Also, we would need to add a case and point to LLDKTests.

**Comment by mphalke** [comment]
> [CLA assistant check](https://cla-assistant.io/analogdevicesinc/PrecisionToolbox?pullRequest=2) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/analogdevicesinc/PrecisionToolbox?pullRequest=2) before we can accept your contribution.
> 
> **Ionut.Muthi** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/analogdevicesinc/PrecisionToolbox?pullRequest=2) it.

@IonutMuthi please sign the CLA to get this PR accepted

**Pull request closed without merging**